packs/feats/raise-symbol.json

```diff
@@ -24,7 +24,21 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Cleric.RaiseSymbol.RollOptionLabel",
+                "option": "shield-religious-symbol",
+                "predicate": [
+                    "self:shield:equipped",
+                    {
+                        "not": "self:effect:emblazon-armament-shield"
+                    }
+                ],
+                "toggleable": true,
+                "value": true
+            }
```

Comment:
You can just predicate on having a shield raised. Presumably if you have emblazon armament shield and a shield raised it's meant to be used as a symbol.

Reply:
Also we can lose the stored value in this

Reply:
> You can just predicate on having a shield raised. Presumably if you have emblazon armament shield and a shield raised it's meant to be used as a symbol.

This is precisely meant to account for someone that is wielding a religious symbol shield that isn't their emblazon armament (they may not even have the feat). The feat itself is open to that as written, although may not be worth accounting for?

> Also we can lose the stored value in this

Oops.